targets/targets.json

```diff
@@ -1349,7 +1349,7 @@
         "core": "Cortex-M4F",
         "extra_labels_add": ["STM32L4", "STM32L476xG", "STM32L476VG"],
         "detect_code": ["0820"],
-        "macros_add": ["USBHOST_OTHER"],
+        "macros_add": ["USBHOST_OTHER", "PAL_USE_HW_ROT=1"],
         "device_has_add": ["ANALOGOUT", "CAN", "LOWPOWERTIMER", "SERIAL_FC", "TRNG", "FLASH"],
```

Comment:
@MarceloSalazar Under \mbed-cloud-client\mbed-client-pal\Source\Port\Reference-Impl\OS_Specific\mbedOS\Board_Specific\, there're 2 following child folders originally. 
TARGET_K64F\pal_plat_K64F.cpp
TARGET_NUCLEO_F429ZI\pal_plat_NUCLEO_F429ZI.cpp
Above 2 source files just include implementation of palStatus_t pal_plat_osGetRoT128Bit(uint8_t *keyBuf, size_t keyLenBytes) for each.  I added a new child folder for L476VG. after getting your comments yesterday. I'm holding the PR.
**TARGET_DISCO_L476VG\pal_plat_DISCO_L476VG.cpp**